tools/modules/framework/module_data_files.sh

```
@@ -0,0 +1,391 @@
+
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)